acceptance-tests/upgrade/multi_step_upgrade_mssql_db_test.go

```diff
@@ -0,0 +1,125 @@
+package upgrade_test
+
+import (
+	"csbbrokerpakazure/acceptance-tests/helpers/apps"
+	"csbbrokerpakazure/acceptance-tests/helpers/brokers"
+	"csbbrokerpakazure/acceptance-tests/helpers/random"
+	"csbbrokerpakazure/acceptance-tests/helpers/services"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("MultiStepUpgradeMssqlDBTest", Label("mssql-db", "multi-step"), func() {
+	When("upgrading broker version", func() {
+		It("should continue to work", func() {
+			By("pushing latest released broker version")
+			serviceBroker := brokers.Create(
+				brokers.WithPrefix("csb-srvdb"),
+				brokers.WithSourceDir(releasedBuildDir),
+			)
+			defer serviceBroker.Delete()
+
+			By("creating a service")
+			serverConfig := newDatabaseServer()
+			serverInstance := services.CreateInstance(
```

Comment:
thoughts: I guess we created this service as a reinforcement that not only a database supports the migration, but the server also

Reply:
I just copied from the mssql-db test. We create a server there as well, as the mssql-db offering needs a server as a parameter. It might be that we already have a pre-created one in the environment and this is redundant, we can think of cleaning this up to speed the test up and reduce cost if that is the case. Will take a look and make the change in a separate PR if that's the case.